Dockerfile

```diff
@@ -113,7 +117,8 @@ RUN rm -vf /charts/*.sh /charts/*.md
 # This image includes any host level programs that we might need. All binaries
 # must be placed in bin/ of the file image and subdirectories of bin/ will be flattened during installation.
 # This means bin/foo/bar will become bin/bar when rke2 installs this to the host
-FROM rancher/hardened-kubernetes:v1.21.7-rke2r2-build20211206 AS kubernetes
+
```

Comment:
Why the extra newline?

I think we also need to bump the image in the rke2-kube-proxy chart, and then bump the chart version up above.

Reply:
https://github.com/rancher/rke2-charts/pull/196 just bumped the chart, can you have a look?

Reply:
bump and remove line https://github.com/rancher/rke2/pull/2171/commits/c229aa7a368714892b692352324a38eb0e94a316

Reply:
ah build failed because it could not fin chart https://drone-pr.rancher.io/rancher/rke2/2658/1/2 
Should pass after https://github.com/rancher/rke2-charts/pull/196 is merged